app/config/AppConfig.scala

```diff
@@ -105,6 +105,13 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
     "cy" -> Lang("cy")
   )
 
+  private def traderKnownFactsReferenceDataService: String =
+    if (isEnabled(StubGetTraderKnownFacts)) {
+      servicesConfig.baseUrl("emcs-tfe-reference-data-stub")
+    } else {
+      servicesConfig.baseUrl("emcs-tfe-reference-data")
```

Comment:
[Optional] this can call this referenceDataService

Reply:
The 'emcs-tfe-reference-data' String?